buildozer/targets/android.py

```diff
@@ -877,10 +877,11 @@ def build_package(self):
         version = self.buildozer.get_version()
 
         # add extra libs/armeabi files in dist/default/libs/armeabi
-        # (same for armeabi-v7a, x86, mips)
+        # (same for armeabi-v7a, arm64-v8a, x86, mips)
         for config_key, lib_dir in (
             ('android.add_libs_armeabi', 'armeabi'),
             ('android.add_libs_armeabi_v7a', 'armeabi-v7a'),
+            ('android.add_libs_arm64_v8a', 'arm64-v8a'),            
```

Comment:
These trailing whitespaces are breaking the linter https://travis-ci.com/kivy/buildozer/builds/118662050
```
buildozer/targets/android.py:884:57: W291 trailing whitespace
```
We need to keep an eye on the CI